bin/ingest_archive_video.py

```diff
@@ -120,19 +143,26 @@ def update_xml(data_dir, collection_id, extention, xml_tree):
 @click.option(
     '-v',
     '--video_dir',
-    default='/Users/xinruyan/Dropbox/naacl-2013/',
     help='Directory contains all videos need to be ingested',
 )
-def main(video_dir):
-    collection_ids = get_collection_ids(video_dir=video_dir)
+@click.option(
+    '-s',
+    '--anth_id_style',
+    default='new',
```

Comment:
You don't have to pass this information; Instead, just get the Anthology ID from the file name, and call `anthology.utils.is_newstyle_id()` ([link](https://github.com/acl-org/acl-anthology/blob/5f0e68184f61d175607aa732c99da995c4f02427/bin/anthology/utils.py#L37)).

Ideally you should refactor the code to handle this—you can infer it and then avoid having to pass it all over.